### PR TITLE
transmission:transmission-web-control initial release

### PR DESCRIPTION
**Pull request by FujiZ**
This mod provides a web UI called Transmission Web Control for transmission. It was once removed when transmission was upgraded to v4.0.

Ref #723

### CODE REVIEW

File: root/etc/s6-overlay/s6-rc.d/init-mod-transmission-transmission-web-control-install/up
Comment (aptalca) [inline root/etc/s6-overlay/s6-rc.d/init-mod-transmission-transmission-web-control-install/up:1]: This needs to point to the actual run file

Comment (FujiZ) [reply root/etc/s6-overlay/s6-rc.d/init-mod-transmission-transmission-web-control-install/up:1]: Done

**Comment by aptalca** [comment]
Can you please test the pushed mod and confirm it works? Then I'll merge

Thanks

**Comment by FujiZ** [comment]
> 

Yes, I have tested it on my PC.
图片

**Pull request merged**